ucey-star/ID8
# Add GitHub Action to randomly assign a PR reviewer

## Description

This PR introduces a GitHub Action to randomly assign a reviewer when a pull request (PR) is created. The action leverages a custom script to select a random reviewer from a predefined list of GitHub usernames and automatically assigns that user to the PR.

The reason for implementing this feature is to ensure an even distribution of PR review responsibilities within the team, encouraging collaboration and preventing bottlenecks where the same reviewer is overloaded. Additionally, the use of this automation reduces the need for manual intervention in the PR review assignment process.

Changes include:

- A GitHub Action file (assign-reviewer.yml) located in .github/workflows/, which triggers when a new pull request is opened.
- A script to randomly select a reviewer from an array of team members.

## Type of Change


## Testing Instructions

To test this:
- After you make a PR, check if it got randomly assigned to a member of the team

## Screenshots (if applicable)

If your changes include visual components, add screenshots showing the affected pages or elements.
